### PR TITLE
Add section on how to test isolated queries and mutations

### PR DESCRIPTION
**Pull request by ospfranco**

### CODE REVIEW

File: app/pages/docs/testing.mdx [hunk 1]
```diff
@@ -106,3 +106,83 @@ our
...
+      id: user.id,
+      role: user.role
+    })
+  }
+
+  return { req: mockReq, res: mockRes, ctx: mockRes.blitzCtx }
+}
+```
+Later in your tests you can use like this:
+
```
Comment (beerose) [inline app/pages/docs/testing.mdx:163]:
```suggestion
```

Later in your tests, you can use it like this:

```

Comment (beerose) [reply app/pages/docs/testing.mdx:163]: Bad formatting, but I meant adding a newline before the `Later in your tests...`, and adding `it` to `you can use **it** like this:`

Comment (ospfranco) [reply app/pages/docs/testing.mdx]: oopsie, you are right, thanks!

**Comment by beerose** [comment]
@ospfranco could you also fix the formatting? 🙏 So that the deployment passes

**Comment by ospfranco** [comment]
Screen Shot 2022-01-04 at 11 26 43

How can I check formatting? `yarn lint` says everything is fine on my side

**Comment by beerose** [comment]
Ok, looks like you only need to add a heading. Here's the error:
```

Error: This heading is missing a handle:
--
21:19:54.477 | Isolated mutation/query testing

```

**Comment by ospfranco** [comment]
done

**Review by beerose — APPROVED** [review]

**Pull request merged**